index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+   <meta charset="UTF-8">
+   <title>To do list</title>
+   <meta name="viewport" content="width=device-width,initial-scale=1">
+   <style></style>
```

Comment:
let's remove the style tag. We don't need it because we store our styles in a separate file